src/components/apps/listing/TableView.js

```diff
@@ -0,0 +1,240 @@
+                                                    tableId,
+                                                    appName,
+                                                    ids.checkbox
+                                                )}
+                                                inputProps={{
+                                                    "aria-label": formatMessage(
+                                                        intl,
+                                                        "ariaCheckbox",
+                                                        {
+                                                            label: appName,
+                                                        }
+                                                    ),
+                                                }}
+                                            />
+                                        </TableCell>
+                                        <TableCell
+                                            padding="none"
+                                            id={build(
+                                                rowId,
+                                                ids.APP_STATUS_ICON
+                                            )}
+                                        >
+                                            <AppStatusIcon
+                                                isPublic={app.is_public}
+                                                isBeta={app.beta}
+                                                isDisabled={app.disabled}
+                                            />
+                                        </TableCell>
+                                        <TableCell>
+                                            <AppName
+                                                baseDebugId={build(
+                                                    rowId,
+                                                    ids.APP_NAME
+                                                )}
+                                                isDisabled={app.disabled}
+                                                name={app.name}
+                                            />
+                                        </TableCell>
+                                        <TableCell
+                                            id={build(
+                                                rowId,
+                                                ids.integratorName
+                                            )}
+                                        >
+                                            {app.integrator_name}
+                                        </TableCell>
+                                        <TableCell
+                                            id={build(rowId, ids.RATING)}
+                                        >
+                                            <Rate
```

Comment:
As an aside, you had mentioned before you left that you had some new ideas on what to do with app ratings but we never discussed them.